vm/kvm-unit-tests/runtest.sh

```diff
@@ -165,6 +165,45 @@ cat test.log | tee -a $OUTPUTFILE
 # check for any new failures
 grep FAIL test.log >> failures.txt
 
+# Add Advanced VIRT repo and update QEMU version
+cat >/etc/yum.repos.d/rhel8-advvirt.repo <<EOF
+[rhel8-advvirt]
+name=rhel8-advvirt
+baseurl=http://download.devel.redhat.com/rhel-8/rel-eng/ADVANCED-VIRT-8/latest-ADVANCED-VIRT-8.1.0-RHEL-8/compose/Advanced-virt/$(uname -i)/os/
```

Comment:
Is there an external repo we can point to ? If not I have another suggestion where we can enable the repo in kpet-db.

Reply:
hi @mcondotta we have the repo enabled by default using kpet now, so you can remove that part about creating the yum repo, and instead enable/disable the repo before/after testing. This is enabled only for rhel8 testing, I see this test is also enabled on ark kernel which runs on fedora, not sure if changes are needed there as well ?

Reply:
hi @mcondotta this has been opened for a while, I never saw a response to my last comment, any further updates ? :-)